### PR TITLE
CMS-692: Improve the current JS according to the standards

### PR DESCRIPTION
**Pull request by Flight**
This PR reverts removed files, which were needed. And also fixes some styling issues.

### CODE REVIEW

File: filer/static/filer/js/addons/focal_point.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+'use strict';
```
Comment (FinalAngel) [inline filer/static/filer/js/addons/focal_point.js:1]: you actually might want to copy the structure (comments, modular pattern, docs) from cms: https://github.com/divio/django-cms/blob/develop/cms/static/cms/js/modules/cms.modal.js#L1

File: filer/static/filer/js/addons/focal_point.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            focalPoint.remove();
+            focalPoint = null;
+        };
+
+        function updatePosition(x, y) {
+            $('#id_subject_location')
+                .val(x === undefined ? '' : parseInt(parseInt(x) * ratio) + ',' + parseInt(parseInt(y) * ratio));
+        }
+
+        function image_init() {
```
Comment (FinalAngel) [inline filer/static/filer/js/addons/focal_point.js]: if we refactor, why not module pattern?

File: filer/static/filer/js/addons/focal_point.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            var location = $('#id_subject_location').val();
+            var x, y;
+
+            PAPER_WIDTH = $('#image_container img').width();
```
Comment (FinalAngel) [inline filer/static/filer/js/addons/focal_point.js]: lowercase, is not a constant

File: filer/static/filer/js/addons/focal_point.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+            PAPER_HEIGTH = $('#image_container img').height();
+            $('#image_container').height(PAPER_HEIGTH + 'px');
+
+            // interface
+            ratio = parseFloat($('#image_container img').attr('rel'));
```
Comment (FinalAngel) [inline filer/static/filer/js/addons/focal_point.js]: cache the element

File: filer/static/filer/js/addons/focal_point.js [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+            // read location from form
+            if (location) {
+                x = parseInt(parseInt(location.split(',')[0]) / ratio);
+                y = parseInt(parseInt(location.split(',')[1]) / ratio);
+            }
+            add(x, y);
+            image_loaded = true;
+        }
+        $(document).ready(function () {
```
Comment (FinalAngel) [inline filer/static/filer/js/addons/focal_point.js]: why do we have two document ready?

**Comment by Flight** [comment]
I think i will fully replace/remove this part. It's just temporary fixes for understanding what it does :)

**Comment by FinalAngel** [comment]
LGTM

**Pull request merged**